### ISSUE TITLE
USER ISSUE: Gui isn't ins sync with mouse

### ISSUE THREAD

**Issue by siggjen** [StrangeLoopGames/EcoIssues#18647]
\
**Version:** 0.9.0.3 beta release-78

**Steps to Reproduce:**
Start Game. Try to click on anything. You have to be below and at the right to hit it

**Expected behavior:**
To click on a button the mouse it over not away from

**Actual behavior:**
None

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods, I just downloaded and installed from Steam

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@siggjen Thank you for the report.
Can you provide a little more information about your problem? Feel free to attach video files or screenshots, and don't forget about the log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
and, please, provide your PC specification (include Operating System).

**Comment by siggjen** [comment]
Hello

I have attached a screenshot it shows the mousepointer to the left of and below the resolution selector box. That where it need to be to open the dropdown selector for resolution. I can't choose Apply as the mouse pointer isn't going enough to the right. My plan was to try to use windowed setting and a decent resolution that is playable. 
There was an upgrade available when I was taking the screenshot so I applied this first. 
OS is Manjaro Linux. CPU i7 7700K, GTX 2080TI 3 screens 4k@60Hz

[player.tar.gz](https://github.com/StrangeLoopGames/EcoIssues/files/5348116/player.tar.gz)
![Screenshot from 2020-10-08 15-32-34](https://user-images.githubusercontent.com/8385246/95465898-fae6f700-097b-11eb-883b-71fee10d114a.png)

**Comment by redwyre** [comment]
You can try pressing Alt+Enter to go into windowed mode.

**Comment by siggjen** [comment]
I managed to figure it out. I had fraction scaling of resolution instead of 3840x2160 which is 4k native. I presume Eco couldn't calculate how to handle mouse pointer in this case. I changed to the native 4k resolution and changed to windowed and then the pointer works perfectly even when I changed back to a fraction scaled resolution. Your keyboard shortcut would of course have been helpful if I had it earlier, thank you.
